compiler/src/iree/compiler/Codegen/Common/test/tile_and_distribute_max_workgroups.mlir

```diff
@@ -0,0 +1,68 @@
+// RUN: iree-opt --pass-pipeline='builtin.module(hal.executable(hal.executable.variant(iree-codegen-tile-and-distribute-to-workgroups{max-workgroup-parallel-dims=1})), canonicalize, cse)' --split-input-file %s | FileCheck %s
```

Comment:
nit: you could add this directly in `tile_and_distribute_to_workgroups.mlir` with a different prefix instead of having two files for this